### ISSUE TITLE
Game disconnects

### ISSUE THREAD

**Issue by murphyonehundred** [shalzuth/LostArkLogger#115]
Game keeps disconnecting and crashing occasionally while Logger is running in the background since the maintenance on Aug 31st. Mostly occurred inside of Legion Raids and once during Challenge Guardian. Disconnects have not been happening without the Logger running so I'm thinking it's some weird EAC interaction with it

**Comment by Ap0k4lyps3** [comment]
Easy Anti-Cheat: Forbidden Tool Detected

Rename window title to something random, will test more later

**Comment by zoulougang** [comment]
Hey ! Did you guys find a solution to use dps meter again? :)

**Comment by murphyonehundred** [comment]
I haven't tried it again ever since the crashes. It was a bit annoying to constantly DC in the middle of raids especially cos the game takes 10 years to launch

**Comment by zoulougang** [comment]
Just did the update, and still got DC with forbidden tool :(

**Comment by Armag06** [comment]
Indeed, the game disconnect, saying a forbbiden tool has been detected. Could you please maybe try to rename the program ?

**Comment by Rootkit-** [comment]
renamed and didn't fix 
![image](https://user-images.githubusercontent.com/5389279/196550278-bb4d46d9-28b8-461d-9b25-d8d3fcf9cf0e.png)

**Comment by Rootkit-** [comment]
Edit: I had rar file of the lostarklogger open and I'll try again later but with the app closed that seemed to also close it which makes me think renaming worked.
